charts/tempo-distributed/README.md.gotmpl

```diff
@@ -20,6 +20,12 @@ helm repo add grafana https://grafana.github.io/helm-charts
 
 A major chart version change indicates that there is an incompatible breaking change needing manual actions.
 
+### From Chart versions < 0.9.0
+
+This release the component label was shortened to be more aligned with the Loki-distributed chart and the [mixin](https://github.com/grafana/tempo/tree/master/operations/tempo-mixin) dashboards.
+
+Please delete the deployments and statefulsets before executing this upgrade.
```

Comment:
i don't disagree with these changes. the shorter names would definitely be clearer. is it common to require completely removing deployments/statefulsets before upgrading? that's my only concern here.

Reply:
Don't know, but I don't see another way to fix this 😞 @unguiculus Do you have any insight regarding these kind of upgrades?

Reply:
I changed labels for Promtail a while ago. Upgrade options are documented here:
https://github.com/grafana/helm-charts/tree/main/charts/promtail#from-chart-versions--300

Reply:
Thanks! will update the README either today or tomorrow.